### PR TITLE
feat: Add "Import document" menu item to document menu

### LINKED ISSUE #1940: Add "Import document" menu item to document menu
Currently the only way to import inside another document is to drag a file onto it in the sidebar, this has confused some folks who thought it wasn't possible. We should add an "Import document…" option in the document menu when the user has permissions to create a child document.

### PR DESCRIPTION
**Pull request by iamsaumya**
closes #1940

### CODE REVIEW

**Review by tommoor — APPROVED** [review]
You're taking all the issues before I can get to them 😆

File: app/menus/DocumentMenu.js [hunk 1]
```diff
@@ -243,6 +309,11 @@ function DocumentMenu({
               }),
               visible: !!can.createChildDocument,
             },
+            {
+              title: t("Import document"),
```
Comment (tommoor) [inline app/menus/DocumentMenu.js:313]: I feel like I want to clarify this menu item but can't quite figure out the wording, perhaps

"Import nested document"

cc @thenanyu

**Comment by iamsaumya** [comment]
> You're taking all the issues before I can get to them 😆

Tackling the small issues so you can focus on the bigger ones 😄

File: app/menus/DocumentMenu.js (hunk 1 above)
Comment (tommoor) [reply app/menus/DocumentMenu.js:313]: Leaving it how it is for now

**Pull request merged**